int32 rigid_bodies
float64[] q_drone